backend/typescript/services/implementations/reviewService.ts

```diff
@@ -386,6 +386,31 @@ class ReviewService implements IReviewService {
 
     return result;
   }
+
+  async updateReviews(id: number, entity: ReviewRequestDTO): Promise<void> {
+    try {
+      const updatedReview = await PgReview.update(
+        {
+          body: entity.body,
+          byline: entity.byline,
+          featured: entity.featured,
+          books: entity.books,
+          tags: entity.tags,
+          updatedAt: Date.now(),
+        },
+        { where: { id }, returning: true },
+      );
+
+      if (updatedReview[0] < 1) {
+        throw new Error(`id ${id} not found`);
```

Comment:
Also see if it is possible to return the updated review - ideally without making another function call but if not, then it should be ok too